src/python/library/tritonclient/http/__init__.py

```diff
@@ -668,10 +669,10 @@ def load_model(self,
             if "parameters" not in load_request:
                 load_request["parameters"] = {}
             load_request["parameters"]["config"] = config
-        for path, content in encoded_files.items():
+        for path, content in files.items():
             if "parameters" not in load_request:
                 load_request["parameters"] = {}
-            load_request["parameters"][path] = content
+            load_request["parameters"][path] = base64.b64encode(content)
```

Comment:
Doesn't `InferenceServerHttpClient::LoadModel()` call `Base64Encode` on the files internally?

Reply:
The Python client library is independent from the C client library, so both need to perform base64 encoding internally.